platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -2984,41 +3161,21 @@ namespace platoon_strategic_ihp
             return true;
         }
 
-        // read status data
+      // read status data
         double current_progress = wm_->routeTrackPos(current_loc).downtrack;
         double speed_progress = current_speed_;
         ros::Time time_progress = ros::Time::now();
 
         // ---------------- use IHP platoon trajectory regulation here --------------------
-        double total_maneuver_length;
-        double target_speed;
-
-        // TODO: Disable temporarily. In future, either remove or replace with speed based gap regulation
-        // // 1. determine if follower 
-        // if (pm_.getNumberOfVehicleInFront() > 0)
-        // {
-        //     // 2. Use IHP platoon trajectory regulation for followers.
-        //     double dt = config_.time_step;
-        //     total_maneuver_length = pm_.getIHPDesPosFollower(dt);
-        //     target_speed = (total_maneuver_length - current_progress) / dt;
-        // }
-        // else
-        // {
-        //     // 3. Use CARMA planning method (control_length = step_time * speed limit).
-        //     target_speed = findSpeedLimit(current_lanelet);   //get Speed Limit TOTO update
-        //     total_maneuver_length = current_progress + config_.time_step * target_speed;
-        // }
-        // ----------------------------------------------------------------
-
-        target_speed = findSpeedLimit(current_lanelet);   //get Speed Limit TOTO update
+        // Note: The desired gap will be adjusted and send to control plugin (via platoon_info_msg) where gap creation will be handled.
+        double total_maneuver_length; 
```

Comment:
total_maneuver_length doesn't appear to be used below.  Does it need to be declared?